packages/lib/src/equals/deepEquals.ts

```diff
@@ -1,3 +1,34 @@
+// 기본 타입, 배열, 객체, 중첩 구조를 정확히 비교
+
 export const deepEquals = (a: unknown, b: unknown) => {
-  return a === b;
+  // 기본 타입
+  if (a === b) return true;
```

Comment:
```suggestion
  if (Object.is(a, b)) return true;
```
Object.is를 이용한 비교가 더 좋을 것 같아요!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Object/is

> Object.is()는 [===](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Strict_equality) 연산자와도 같지 않습니다. Object.is()와 ===의 유일한 차이는 부호 있는 0과 NaN 값들의 처리입니다. === 연산자(및 == 연산자)는 숫자값 -0과 +0을 같게 처리하지만, [NaN](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/NaN)은 서로 같지 않게 처리합니다.